packages/web-ui/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "extends": "@latitude-data/typescript-config/base.json",
   "compilerOptions": {
     "jsx": "react-jsx",
-    "moduleResolution": "NodeNext",
-    "module": "NodeNext",
+    "moduleResolution": "Bundler",
+    "module": "ESNext",
```

Comment:
Let's review this decision carefully before merging. I know you did to make it work but let's thing deeper why this is necessary